daemon/cluster/noderunner.go

```diff
@@ -271,7 +271,13 @@ func (n *nodeRunner) enableReconnectWatcher() {
 		if n.stopping {
 			return
 		}
-		config.RemoteAddr = n.cluster.getRemoteAddress()
+		remotes := n.cluster.getRemoteAddressList()
+		if len(remotes) > 0 {
+			config.RemoteAddr = remotes[0]
+		} else {
+			config.RemoteAddr = ""
+		}
```

Comment:
the previous behavior of the getRemoteAddress was to return one IP or "". In this case I reimplemented the same logic. Maybe @tonistiigi  can shed a light on it.
The only case that I was able to think about for returning "" is when there is only 1 manager and this method is called on that manager